dbft.go

```diff
@@ -337,24 +337,21 @@ func (d *DBFT[H]) onPrepareRequest(msg ConsensusPayload[H]) {
 }
 
 func (d *DBFT[H]) processMissingTx() {
-	missing := make([]H, 0, len(d.TransactionHashes)/2)
-
 	for _, h := range d.TransactionHashes {
 		if _, ok := d.Transactions[h]; ok {
 			continue
 		}
 		if tx := d.GetTx(h); tx == nil {
-			missing = append(missing, h)
+			d.MissingTransactions = append(d.MissingTransactions, h)
```

Comment:
I thought about that, this change in fact keeps the old behavior since `missing` and `d.MissingTransactions` ended up being the same slice anyway. But documentation can be improved, true.